11LessonEleven/homework/inspired.html

```diff
@@ -3,12 +3,31 @@
 
 <head>
     <title>Inspired</title>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Bree+Serif|Open+Sans" rel="stylesheet">
+    <link rel="stylesheet" href="css/css-reset.css" />
+    <link rel="stylesheet" href="css/inspired.css">
 </head>
 
 <body>
-    <main>
-
+    <main class="hero">
+        <div class="inspiration">
+            <h1>Let your life be an Inspiration</h1>
+            <p>Tart soufflé pudding bear claw marshmallow caramels biscuit gingerbread topping. Candy cupcake chocolate cake.</p>
+        </div>
     </main>
+    <div class="words clearfix">
+        <article class="blog">
+            <h1>Latest News From Our Blog</h1>
+            <p>Macaroon chocolate bar croissant tiramisu jelly sweet topping gummies cookie. Macaroon halvah sugar plum tart gummi bears fruitcake sugar plum. Pastry biscuit fruitcake wafer carrot cake jujubes cake.</p>
+        </article>
+        <article class="mumbai">
+            <h1>Everyday Food For Mumbai Children</h1>
+            <p>Macaroon chocolate bar croissant tiramisu jelly sweet topping gummies cookie. Macaroon halvah sugar plum tart gummi bears fruitcake sugar plum. Pastry biscuit fruitcake wafer carrot cake jujubes cake.</p>
+        </article>
+    </div>
 </body>
 
-</html>
\ No newline at end of file
+</html>

```

Comment:
Great job! Just need to change some font sizes in your phone media queries.
<img width="607" alt="screen shot 2016-09-13 at 6 04 57 pm" src="https://cloud.githubusercontent.com/assets/12698279/18494620/db86b374-79dc-11e6-87cf-28756b17a391.png">